frankDoc/src/main/java/nl/nn/adapterframework/frankdoc/model/AttributeValue.java

```diff
@@ -0,0 +1,60 @@
+/* 
+Copyright 2021 WeAreFrank! 
+
+Licensed under the Apache License, Version 2.0 (the "License"); 
+you may not use this file except in compliance with the License. 
+You may obtain a copy of the License at 
+
+    http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software 
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+
+package nl.nn.adapterframework.frankdoc.model;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.Logger;
+
+import lombok.Getter;
+import nl.nn.adapterframework.doc.EnumLabel;
+import nl.nn.adapterframework.frankdoc.doclet.FrankAnnotation;
+import nl.nn.adapterframework.frankdoc.doclet.FrankDocException;
+import nl.nn.adapterframework.frankdoc.doclet.FrankEnumConstant;
+import nl.nn.adapterframework.util.LogUtil;
+
+public class AttributeValue {
```

Comment:
Als dit alleen om Enums of lijsten gaat dan moet dat uit de classnaam blijken

Reply:
We hadden al AttributeValues en AttributeValuesFactory. Die classes moeten eigenlijk ook hernoemd worden. Als ik dat doe, dan wordt de pull request groter. Gerrit en Niels, willen jullie dat? We kunnen ook een aparte pull request doen voor de renames.

Reply:
Ik heb de class hernoemd. We hebben nu AttributeEnumFactory, AttributeEnum en AttributeEnumValue.